src/orch/orch.c

```diff
@@ -1,17 +1,29 @@
 #include "../common/dbus.h"
+#include "../ini/ini.h"
 #include "controller.h"
 #include "opt.h"
 
 #include <stdio.h>
 #include <stdlib.h>
 
+/* A function that takes care of printing the lines of the ini file  */
+static int ini_handler_func(void *user, const char *section, const char *name, const char *value) {
+        printf("[%s] %s = %s\n", section, name, value);
+        return EXIT_FAILURE;
+}
+
 int main(int argc, char *argv[]) {
         fprintf(stdout, "Hello from orchestrator!\n");
 
         int port;
-        get_opts(argc, argv, &port);
+        char *configPath = calloc(1024, sizeof(char));
+
+        int r = get_opts(argc, argv, &port, configPath);
+        if (r < 0) {
+                fprintf(stderr, "Failed to parse command-line arguments: %s\n", strerror(-r));
```

Comment:
We are already "logging" the error in `get_opts`, making this "log" unnecessary. So it can be removed. 

In addition, `strerror` searches an internal error array and maps the int param to a description of that error code (see [here](https://www.tutorialspoint.com/c_standard_library/c_function_strerror.htm))... haven't found a reference to the error array, unfortunately. Not sure what the exit codes for success/failure map to, but does not add much information.